floss/render/default.py

```diff
@@ -129,6 +144,20 @@ def strtime(seconds):
     return f"{m:02.0f}:{s:02.0f}"
 
 
+def render_gostrings(language_strings, language_missed_strings, console, verbose, disable_headers):
+    strings = sorted(list(language_strings) + language_missed_strings, key=lambda s: s.offset)
+    render_heading("FLOSS GO STRINGS", len(strings), console, verbose, disable_headers)
+    for s in strings:
+        if verbose == Verbosity.DEFAULT:
+            console.print(s.string, markup=False)
+        else:
+            colored_string = string_style(s.string)
+            console.print(
+                f"0x{s.offset:>08x} {'go' if any([ls == s for ls in language_strings]) else '  '} {colored_string}"
```

Comment:
this is helpful during development, but do we want to continue using an extra column to indicate a language-specific extracted string?